src/main/java/uk/gov/hmcts/divorce/citizen/notification/RespondentSolicitorReminderNotification.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.divorce.citizen.notification;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.divorcecase.model.Applicant;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.notification.ApplicantNotification;
+import uk.gov.hmcts.divorce.notification.CommonContent;
+import uk.gov.hmcts.divorce.notification.NotificationService;
+
+import java.util.Map;
+
+import static org.apache.commons.lang3.StringUtils.isNotEmpty;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.ISSUE_DATE;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.NOT_PROVIDED;
+import static uk.gov.hmcts.divorce.notification.CommonContent.IS_DISSOLUTION;
+import static uk.gov.hmcts.divorce.notification.CommonContent.IS_DIVORCE;
+import static uk.gov.hmcts.divorce.notification.CommonContent.NO;
+import static uk.gov.hmcts.divorce.notification.CommonContent.SIGN_IN_URL;
+import static uk.gov.hmcts.divorce.notification.CommonContent.SOLICITOR_NAME;
+import static uk.gov.hmcts.divorce.notification.CommonContent.SOLICITOR_REFERENCE;
+import static uk.gov.hmcts.divorce.notification.CommonContent.SUBMISSION_RESPONSE_DATE;
+import static uk.gov.hmcts.divorce.notification.CommonContent.YES;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.RESPONDENT_SOLICITOR_HAS_NOT_RESPONDED;
+import static uk.gov.hmcts.divorce.notification.FormatUtil.DATE_TIME_FORMATTER;
+
+@Slf4j
+@Component
+public class RespondentSolicitorReminderNotification implements ApplicantNotification {
+
+    @Autowired
+    private NotificationService notificationService;
+
+    @Autowired
+    private CommonContent commonContent;
+
+    @Override
+    public void sendToApplicant2Solicitor(CaseData caseData, Long caseId) {
+        log.info("Sending the respondent solicitor has not responded notification to the solicitor for case : {}", caseId);
+
+        notificationService.sendEmail(
```

Comment:
what about cases where we don't have app2 solicitor email ?

Reply:
added email check condition in SystemRemindRespondentSolicitor